supplementary_style_guide/glossary_terms_conventions/general_conventions/p.adoc

```diff
@@ -57,6 +57,17 @@
 
 *See also*:
 
+[[peer-pod]]
+==== image:images/yes.png[yes] peer pod (noun)
+*Description*: In Red{nbsp}Hat OpenShift sandboxed containers, a _peer pod_ extends the concept of a standard pod. Unlike a standard sandboxed container, where the virtual machine is created on the worker node itself, in a peer pod, the virtual machine is created through a remote hypervisor using any supported hypervisor or cloud provider API.
```

Comment:
1. Consider putting each sentence on a separate line.
2. The second sentence is very long, perhaps we should split it into two separate sentences? 
3. The text refers to "standard pod" and "standard ... container". Do these terms refer to the same thing?
4. Is "through" (a remote hypervisor) the correct term?
5. Change "using" to "by using", if that's what's meant.
6. Repeat "API" after "any supported hypervisor", if that's what's meant.

Reply:
The "one sentence per line" guideline is not followed elsewhere in this file.